sdks/python/apache_beam/pvalue.py

```diff
@@ -329,8 +329,9 @@ def _from_runtime_iterable(it, options):
     elif len(head) == 1:
       return head[0]
     raise ValueError(
-        'PCollection with more than one element accessed as '
-        'a singleton view.')
+        'PCollection of length %d with more than one element accessed as a '
```

Comment:
<!--new_thread; commit:306a1dc4a1e072a65c739e179b5e1017e2069709; resolved:0-->
Let's use "size" instead of "length" as PCollections are unordered.